meson.build

```diff
@@ -253,8 +253,11 @@ install_headers(
     'xkbcommon/xkbcommon-names.h',
     subdir: 'xkbcommon',
 )
+
+# This variable may be used to build as a subproject and should not be renamed.
```

Comment:
fwiw, an approach I've used in another project (can't remember which one) is to have a specific section in the `meson.build` file to collect these "APIs", even with just simple aliases, i.e. around these lines:
```
# The below variables are considered API for subprojects
libxkbcommon_dep = dep_libxkbcommon
libxkbcommon_x11_dep =  dep_libxkbcommon_x11
```
The main benefit here is obviously to make it easier to keep these stable and have them in a single location.

Reply:
I like this idea, will do it as a follow up.